nengo_ocl/tests/conftest.py

```diff
@@ -4,24 +4,11 @@
 import pyopencl as cl
 import pytest
 
-import nengo_ocl
-
 from nengo.conftest import plt, rng  # noqa: F401
 
 
-# --- Ensure all Simulators have same context
 ctx = cl.create_some_context()
-
-
-class OclSimulator(nengo_ocl.Simulator):
-    def __init__(self, *args, **kwargs):
-        super(OclSimulator, self).__init__(*args, context=ctx, **kwargs)
-
-
 @pytest.fixture(scope="session")
-def Simulator(request):
```

Comment:
Removing this affects `nengo_ocl/tests/test_simulator.py` and `nengo_ocl/tests/test_planners.py`. @tbekolay, can you take a look?